test/helpers.c

```diff
@@ -0,0 +1,38 @@
+/* This include order verifies that we can include `pcap/bpf.h` before `net/bpf.h` */
+#include "helpers.h"
+#include <net/bpf.h>
+#include <net/bpfjit.h>
+
+#include <assert.h>
+#include <pcap/pcap.h>
```

Comment:
It's slightly confusing that pcap.h is included again here, given the comment 6 lines above this.

Reply:
pcap includes are now put together